serverless/functions/Security.yml

```diff
@@ -0,0 +1,59 @@
+# ================== Security =====================
+
+securityInitialization:
+  handler: security/handler.initialization
+  description: 'Create CA for change refresh token period'
+  tags:
+    client: ${file(env.yml):${opt:stage, self:provider.stage}.CLIENT}
+  timeout: 28
+  environment:
+    CHANGE_REFRESH_TOKEN_PERIOD_CA_URL: !Sub
+      - '${apiBaseUrl}${path}'
+      - apiBaseUrl: ${self:custom.apiBaseURL.${opt:stage, self:provider.stage}}
+        path: api/security/change-refresh-token-period
+  events:
+    - http:

```

Comment:
We need to add some auth stuffs for this method. Everybody from the internet who have this link can use it. Probably, we can use IAM authorization. https://docs.aws.amazon.com/apigateway/latest/developerguide/permissions.html.

Reply:
I think it should be secured in the same way as all CAs and WHs are secured now.

Reply:
As i can see the lambda does not use any auhtorizer. So, it could be called by any person in the internet who has the endpoint.